### ISSUE TITLE
connecting to mongodb error

### ISSUE THREAD

**Issue by whoodat** [askmike/gekko#2204]
When I try running a backtest with mongodb I am receiving the following error:

Gekko encountered an error and can't continue

Error:

{ MongoError: not authorized on gekko to execute command { find: "gdax_candles", filter: { pair: "USD_ETH" }, $db: "gekko" }
    at Function.MongoError.create (/home/whoodat/gekko/node_modules/mongodb-core/lib/error.js:31:11)
    at queryCallback (/home/whoodat/gekko/node_modules/mongodb-core/lib/cursor.js:212:36)
    at /home/whoodat/gekko/node_modules/mongodb-core/lib/connection/pool.js:469:18
    at _combinedTickCallback (internal/process/next_tick.js:131:7)
    at process._tickCallback (internal/process/next_tick.js:180:9)
  name: 'MongoError',
  message: 'not authorized on gekko to execute command { find: "gdax_candles", filter: { pair: "USD_ETH" }, $db: "gekko" }',
  ok: 0,
  errmsg: 'not authorized on gekko to execute command { find: "gdax_candles", filter: { pair: "USD_ETH" }, $db: "gekko" }',
  code: 13,
  codeName: 'Unauthorized' } '\n\n'

Meta debug info:

Gekko version: v0.5.14
Nodejs version: v8.10.0

Has anyone run into this issue?

**Comment by whoodat** [comment]
Got it to work for 3.2MongoDB guess I will wait for new gekko code to upgrade to 3.6

**Comment by askmike** [comment]
yes, mongo support is on the list for the new gekko :) Open to PRs in the meanwhile.

**Comment by whoodat** [comment]
Hi Mike,

I recently did a git pull on the develop branch and now I am not able to use the mongodb again. I did not upgrade mongodb...left at version 3.2 and now am receiving the following error.

Gekko encountered an error and can't continue

Error:

{ MongoError: not authorized on gekko to execute command { find: "gdax_candles", filter: { pair: "USD_ETH" } }
    at Function.MongoError.create (/home/whoodat/gekko/node_modules/mongodb-core/lib/error.js:31:11)
    at queryCallback (/home/whoodat/gekko/node_modules/mongodb-core/lib/cursor.js:212:36)
    at /home/whoodat/gekko/node_modules/mongodb-core/lib/connection/pool.js:469:18
    at _combinedTickCallback (internal/process/next_tick.js:131:7)
    at process._tickCallback (internal/process/next_tick.js:180:9)
  name: 'MongoError',
  message: 'not authorized on gekko to execute command { find: "gdax_candles", filter: { pair: "USD_ETH" } }',
  ok: 0,
  errmsg: 'not authorized on gekko to execute command { find: "gdax_candles", filter: { pair: "USD_ETH" } }',
  code: 13 } '\n\n'

Meta debug info:

Gekko version: v0.5.14
Nodejs version: v8.10.0

whoodat@whoodatserver:~/gekko$ mongo --version
MongoDB shell version: 3.2.20

Any ideas?
